### ISSUE TITLE
MCQ: JAWS selecting on radio options

### ISSUE THREAD

**Issue by joannecas** [adaptlearning/adapt_framework#3010]
### Subject of the issue/enhancement/features
When using JAWS, single option/radio mcq items are not selectable and the tabbing is trapped in the options. Works fine in NVDA. Needed for JAWS usage WCAG 2.0 Level A and higher. Keyboard and Keyboard no trap requirements. 

### Your environment
Appears in both 5.5.1 and 5.7.0 (not tested in latest versions, no existing or closed issue exists)
* Chrome Version 87.0.4280.141 (Official Build) (64-bit)
* Edge Version 87.0.664.66 (Official build) (64-bit)
* MacOS 10.14.6 Parallels V16 Windows 10

### Steps to reproduce
Setup MCQ with _selectable 1, and one true answer. (Testing using parallels on local server). Forward through to question options with JAWS (40 minute trial rebootable available). Enter does not select option. (Works fine with multiple select items and using NVDA). 

### Expected behaviour
Enter should allow the user to select their option for the single selectable item question and tab/forward to submit/auto-submit.

### Actual behaviour
User focuses on their option and presses Enter, no selection is made, user cannot navigate outside of the question items.

### Other notes
No console errors. ```'keyup .js-item-input': 'onKeyPress'``` does not fire

**Comment by moloko** [comment]
@joannecas I can check this tomorrow but it does sound from your description that it might be Jaws going into ‘forms mode’... if you can press Esc to get out of it then it almost certainly is.

**Comment by joannecas** [comment]
@moloko I do believe you are right. Esc does allow exiting the items list. On retest of this theory, if you press Enter the selection is moving with your focus. If you press Enter, your selection does indeed stay on the item, then you can forward to submit or out of the item box backwards. It was not functionality I was expecting, but I am fine with it as long as it is expected behaviour among JAWS users. It might be an easy close for this issue if so!

Thanks @moloko for responding!

**Comment by moloko** [comment]
As far as I'm aware this is standard Jaws behaviour for a 'radiogroup' set of controls. You should experience similar behaviour with the slider component.
https://dequeuniversity.com/screenreaders/jaws-keyboard-shortcuts#jaws-forms
